web/src/components/MetaNode.vue

```diff
@@ -15,20 +15,24 @@
         <v-icon v-else>mdi-circle-medium</v-icon>
       </template>
     </v-select>
-    <v-text-field
+    <!-- changed v-text-field to v-textarea for name and description-->
+    <!-- should I add auto-grow -->
```

Comment:
I think we can leave auto-grow out for now. The name/description shouldn't be long enough to warrant that happening anyway, and if it is it may take up more space than we want. If you agree then we can remove these comments.

Reply:
yes I can remove the comments and push again